### DISCUSSION TITLE
On creating yaml file

### DISCUSSION THREAD

**Discussion by mh-tanver** [AstroBridge/BDLensing#232] [Show and tell]
Here is the colab link of yaml file notebook for my model. Any suggestions for further improvements will be appreciated.

https://colab.research.google.com/drive/1oto17UotnnuS52j-bwnZJ5wZSd7kO1w5?usp=sharing

**Comment by ajshajib** [comment]
I suggest you make a PR, that makes it easier to provide comments on code.

**Reply by mh-tanver** [reply]
Okay Bhaiya, I am opening a PR.
